### PR TITLE
Bugfix/df 1176 big french headers

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/widgets/shared/widget-title/widget-title.component.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<h2 class="panel-title d-flex flex-row">
+  <alv-custom-icon class="panel-icon"
+                   aria-hidden="true"
```
Comment (99999989) [inline alv-portal-ui/src/app/widgets/shared/widget-title/widget-title.component.html]: `aria-hidden` is not needed here. The screen reader will just skip it anyways.

File: alv-portal-ui/src/app/widgets/shared/widget-title/widget-title.component.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Component, Input } from '@angular/core';
+import { IconKey } from '../../../shared/icons/custom-icon/custom-icon.component';
+
+@Component({
+  selector: 'alv-widget-title',
+  templateUrl: './widget-title.component.html',
```
Comment (99999989) [inline alv-portal-ui/src/app/widgets/shared/widget-title/widget-title.component.ts]: Please add `changeDetection: ChangeDetectionStrategy.OnPush`

**Comment by ganqqwerty** [comment]
I decided that it would be better to add only css classes to `link-panel.component.ts`, see new commits

File: alv-portal-ui/src/app/shared/layout/link-panel/link-panel.component.ts [hunk 3]
```diff
@@ -37,9 +37,6 @@ export class LinkPanelComponent implements OnInit {
   @Input()
   numberOfColumns: NumberOfColumns = 2;
 
-  @Input()
-  image: string;
```
Comment (ganqqwerty) [inline alv-portal-ui/src/app/shared/layout/link-panel/link-panel.component.ts]: this were not used anywhere and got deprecated after. Removed

**Comment by ganqqwerty** [comment]
> I looked over your PR and noticed that the fix could be much easier. Just add `display: flex; flex-direction: row;` to the `.panel-title` class of `panel.scss`. What do you think?

Indeed it's simpler. Less code - less struggle.

**Pull request merged**